scripts/lint_russian_rki_metrics.py

```
@@ -0,0 +1,105 @@
+review of explicit-subject bias, agentive-passive density and bookish copulas.
+"""
+from __future__ import annotations
+
+import re
+from typing import Any
+
+CYRILLIC_WORD_RE = re.compile(r"[А-Яа-яЁё-]+")
+INLINE_CODE_RE = re.compile(r"`[^`\n]*`")
+URL_RE = re.compile(r"https?://\S+|www\.\S+", re.I)
+NOMINATIVE_PERSONAL_PRONOUN_RE = re.compile(r"(?<![А-Яа-яЁё-])(?:я|ты|он|она|оно|мы|вы|они)(?![А-Яа-яЁё-])", re.I)
+SHORT_PASSIVE_RE = re.compile(r"(?<![А-Яа-яЁё-])[А-Яа-яЁё]{3,}(?:ан|ян|ен|ён|т)(?:а|о|ы)?(?![А-Яа-яЁё-])", re.I)
```

Ruff (0.16.1)

[warning] String contains ambiguous `А` (CYRILLIC CAPITAL LETTER A). Did you mean `A` (LATIN CAPITAL LETTER A)?

(RUF001)